frappe/automation/workspace/tools/tools.json

```diff
@@ -145,7 +145,7 @@
    "type": "Link"
   }
  ],
- "modified": "2022-08-23 14:42:58.364898",
+ "modified": "2022-01-13 17:48:48.456763",
```

Comment:
I can still change in JSON files (tools.json, build.json, customization.json & website.json)